bluemira/equilibria/flux_surfaces.py

```diff
@@ -455,22 +455,24 @@ def __init__(self, coords: Coordinates):
 
         self.alpha = None
 
-    def clip(self, first_wall: Coordinates):
+    def clip(self, flux_intercepting_surface: Coordinates, verbose: bool = True):  # noqa: FBT001, FBT002
```

Comment:
We dont need extra flags here. The suppression should come from elsewhere.
```suggestion
    def clip(self, flux_intercepting_surface: Coordinates, verbose: bool = True):  # noqa: FBT001, FBT002
```